pkg/services/secrets/kvstore/kvstore.go

```diff
@@ -14,7 +16,12 @@ const (
 	AllOrganizations = -1
 )
 
-func ProvideService(sqlStore sqlstore.Store, secretsService secrets.Service, remoteCheck UseRemoteSecretsPluginCheck) SecretsKVStore {
+func ProvideService(sqlStore sqlstore.Store,
```

Comment:
nit: I noticed you moved all parameters to a new line except `sqlStore sqlstore.Store`, wouldn't be better to have it in a new line as well?